content/kubernetes/memory/node-resources.md

```diff
@@ -6,54 +6,87 @@ weight: 17
+## Eviction thresholds
 
-## Resource quotas
+Kubernetes uses [node-pressure eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/) to terminate pods and reclaim node resources. Eviction thresholds are managed by `kubelet` arguments.
 
-Kubernetes uses the `ResourceQuota` object to limit resource consumption per namespace. This lets you limit the number of objects created by a namespace or the amount of compute resources consumed by a namespace. 
+Best practice for production Redis Enterprise deployments: 
 
-The resource settings for Redis Enterprise for Kubernetes are defined in the `operator.yaml` and the [`RedisEnterpriseCluster`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec) custom resource.
+* Set a high [soft eviction threshold](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#soft-eviction-thresholds), which will change the node condition early and notify the administrator.
+* Set the `eviction-soft-grace-period` high enough for the administrator to scale the deployment up or out when needed.
+* Set the `eviction-max-pod-grace-period` high enough to allow Redis Enterprise pods to migrate the database before being evicted.
 
-The following settings are the minimum workloads for the operator to function.
+## Monitor resources
+
+There are two [node conditions](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) that are `True` if your [eviction threshold](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#eviction-thresholds) has been met and a pod eviction is likely:  `MemoryPressure` and `DiskPressure`.
+
+Use the `kubectl get nodes` command to view these conditions.
 
 ```yaml
-  resources:
-    limits:
-      cpu: 0.5
-      memory: 256Mi
-    requests:
-      cpu: 0.5
-      memory: 256Mi
+> kubectl get nodes -o jsonpath='{range .items[*]}name:{.metadata.name}{"\t"}MemoryPressure:{.status.conditions[?(@.type == "MemoryPressure")].status}{"\t"}DiskPressure:{.status.conditions[?(@.type == "DiskPressure")].status}{"\n"}{end}'
+name:gke-55d1ac88-213c	MemoryPressure:False	DiskPressure:False
+name:gke-55d1ac88-vrpp	MemoryPressure:False	DiskPressure:False
+name:gke-7253cc19-42g0	MemoryPressure:False	DiskPressure:False
 ```
-For more details on using resource quotas, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/policy/resource-quotas/).
 
+## Resource limits
+
+The Redis Enterprise operator is set with resource limits and requests by default. You can find the recommended settings in the []`operator.yaml`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/operator.yaml) file.
```

Comment:
I still suggest to remove the entire "Resource limits" section (see prev comment).
But if we do keep it, need to fix link formatting:

```suggestion
The Redis Enterprise operator is set with resource limits and requests by default. You can find the recommended settings in the [`operator.yaml`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/operator.yaml) file.
```